subprojects/gradle.properties

```diff
@@ -1,6 +1,6 @@
 # Gradle properties are not passed to included builds https://github.com/gradle/gradle/issues/2534
 kotlin.code.style=official
-org.gradle.jvmargs=-Xmx4g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=10 -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx6g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=10 -Dfile.encoding=UTF-8
```

Comment:
is it clear what task causes OOM?

Reply:
I don't know how to find but could reproduce.
I need drop Gradle caches, build folder and sync the project as a first time

Reply:
OOM left hprof file as far as i remember
i think we can look at it's contents for clues